app/src/index.html

```diff
@@ -26,12 +24,13 @@
     </div>
   </live-interview-app>
 
+  <router-outlet></router-outlet>
```

Comment:
`<router-outlet></router-outlet>` is already existing inside root `live-interview-app` component and should be located _only_ there.